labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -61,7 +83,10 @@ object RecursionPatternMatchingExercise {
    * List(1,1,2,3,1,1) -> List((4,1),(1,2),(1,3))
    */
   def amountEqualMembers[T](in: List[T]): List[(Int, T)] = {
-    error("fix me")
+    in match {
+      case List() => List()
+      case s::rest => List((1 + rest.filter((a) => a == s).length, s)) ++ amountEqualMembers(rest.filter((a) => a != s))
```

Comment:
Instead of filter a!=s you could just use filterNot! Doesn't really save much but just a side comment. I think calling count on in like: (in.count ( _  == s), s) will do the same as List((1 + rest.filter((a) => a == s).length, s)) (This is how I did it). Not sure if it's any more readable, but it's definitely shorter.